backend/src/utils/auth/authUtils.ts

```diff
@@ -0,0 +1,48 @@
+import jwt from 'jsonwebtoken';
+
+import {
+  DEFAULT_ACCESS_TIMEOUT,
+  DEFAULT_NO_SECRET,
+  DEFAULT_REFRESH_TIMEOUT,
+} from 'config/constants';
+import { IAccessJwtPayload, IRefreshJwtPayload, ITokens } from 'interfaces/auth/authInterfaces';
+import { IUser } from 'interfaces/db/entities';
+
+const generateJWT = async (userData: IUser): Promise<ITokens> => {
+  try {
+    if (!userData._id) {
+      throw new Error('wrong user id');
+    }
+
+    const { _id: id, role } = userData;
+    const timeout = process.env.JWT_EXPIRATION_TIME_ACCESS || DEFAULT_ACCESS_TIMEOUT;
+    const refreshTimeout = process.env.JWT_EXPIRATION_TIME_REFRESH || DEFAULT_REFRESH_TIMEOUT;
+
+    const accessToken = await jwt.sign({ id, role }, `${process.env.JWT_SECRET}`, {
```

Comment:
I think you should use `|| DEFAULT_NO_SECRET` as you did it in methods below. Just like `const secret = process.env.* || DEFAULT_NO_SECRET`.

Reply:
i think that it won't be secure if we will use default string which will be store in our server constans, so if we don't have secret key we will just get an error and also we can't allow user to input secret keys as it's only deploy responsibility

Reply:
plus it will be hard to debug if we don't have own secret key for server it's still staying true

Reply:
Yeap, you're right.
Resolved.